src/components/ha-date-range-picker.ts

```diff
@@ -457,17 +430,14 @@ export class HaDateRangePicker extends LitElement {
         border-top: 1px solid var(--divider-color);
       }
 
-      ha-textfield {
+      ha-textarea {
         display: inline-block;
-        max-width: 250px;
-        min-width: 220px;
+        width: 340px;
+        margin-right: 8px;
```

Comment:
Instead of this here and at the prev button add: `gap: 8px;` to `.date-range-inputs`

Reply:
ok, sorry missed that: have now changed it